Assets/Scripts/UI/Dialogue/DialogueTrigger.cs

```diff
@@ -8,9 +8,7 @@ public class DialogueTrigger : MonoBehaviour
     [SerializeField] private DialogueManager manager;
 
     void Start() {
-        if (!manager) {
-            manager = FindObjectOfType<DialogueManager>();
-        }
+        if (!manager) manager = FindObjectOfType<DialogueManager>();
```

Comment:
`DialogueManager` can just be made into a singleton to avoid lines like this. See `UIEventListener` for an example.